next/tailwind.config.js

```diff
@@ -104,7 +104,10 @@ module.exports = {
           universal: {
             100: 'var(--universal-gray-100)',
             200: 'var(--universal-gray-200)',
+            300: '#C2C2C2',
             500: 'var(--universal-gray-500)',
+            600: '#5C5C5C',
+            700: '#333333',
```

Comment:
I know some colors are defined here and some outside - and frankly, our styles setup is a mess at the moment

but to start moving it in the right direction:
- let's add any new colors as variables to bratislava.css, so they are (mostly) in one place
- the forms colors may have slightly different color pallete - let's keep them clumped together and keep any new colors added prefixed with 'forms-'
- in the tailwind config as well - keep them clumped together under `forms` subobject (colors -> forms -> gray -> name-of-the-gray-variant)

By the way - the reason colors are defined are as variables is to allow for dynamic switching of color themes in different sections - on some subpages, some of these variables are overridden to provide different pallete. Much to be desired in terms of organisation still, but just to explain this one part :)